dora/src/bytecode/data.rs

```diff
@@ -361,6 +369,14 @@ impl BytecodeFunction {
     pub fn const_pool(&self, idx: ConstPoolIdx) -> &ConstPoolEntry {
         &self.const_pool[idx.to_usize()]
     }
+
+    pub fn offset_position(&self, offset: u32) -> Position {
+        let index = self
+            .positions
+            .binary_search_by_key(&offset, |&(o, _)| o)
+            .expect("position not found");
```

Comment:
If our position-table contains ranges, then I don't think we can use `expect` here since it is okay to not have an exact match. If I am reading the docs right, we should use both Ok(idx) and Err(idx).

Reply:
Changed to assume ranges.